test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+xIFS=$IFS
```

Comment:
csak erdeklodom, hogy ezt mire hasznalod?

Reply:
@hegyhati a kérdés maga teljesen jogos. Több helyen találkoztam ezzel a módszerrel és azt hittem (reméltem), hogy az internal field separator értékének átadása segít majd a másik branchnél felmerült problémánkon debug közben, de mint utólag kiderült nincs funkciója esetünkben, főleg úgy nem, hogy nem is hivatkozunk rá.

Reply:
@hegyhati javítottam!